### DISCUSSION TITLE
WIll "RC" be added to billing_code_type in the in_network_rate schema

### DISCUSSION THREAD

**Discussion by josemedinawork** [CMSgov/price-transparency-guide#348] [Q&A] [answered]
The documentation on GitHub under "Additional Notes Concerning billing_code_type" indicate that billing_code_type allows for the value RC, however, the value is not present in the schema.
"billing_code_type": {
            "enum": [
              "CPT",
              "HCPCS",
              "ICD",
              "MS-DRG",
              "R-DRG",
              "S-DRG",
              "APS-DRG",
              "AP-DRG",
              "APR-DRG",
              "APC",
              "NDC",
              "HIPPS",
              "LOCAL",
              "EAPG",
              "CDT"
            ]
          }

**Comment by shaselton-usds** [comment, marked as answer]
Thanks for the extra set of eyes!

PR opened here: https://github.com/CMSgov/price-transparency-guide/pull/351
